packages/docs/src/repl/worker/app-bundle-client.ts

```diff
@@ -19,7 +19,7 @@ export const appBundleClient = async (
     srcInputs: getInputs(options),
     // Older versions don't support `segment`
     entryStrategy:
-      options.entryStrategy.type === 'segment' ? { type: 'hook' } : options.entryStrategy,
+      options.entryStrategy?.type === 'segment' ? { type: 'hook' } : options.entryStrategy,
```

Comment:
Oh wow, it's not always defined?

Reply:
I had to fix this in order to suppress error from the eslint because it is defined as `entryStrategy?: EntryStrategy;`

Reply:
Weird that TS didn't encounter this problem when it got merged